packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -485,7 +489,9 @@ export abstract class QueryEditorStore {
       );
 
       yield this.setUpEditorState();
-      yield flowResult(this.buildGraph());
+      if (this.requiresGraphBuilding()) {
+        yield flowResult(this.buildGraph());
```

Comment:
with all of this, why don't we just simply **override** `buildGraph()`? the code change is getting a bit wild :)

In the future, if building graph from small cache is the way we go about in majority of the cases, we can introduce a new function to get the cache and default `buildGraph()` to make use of that function

Reply:
I do override buildGraph() for DataSpaceCreatorStore but for ExistingQueryEditorStore I need to use it via extensions.
But i guess creating a new function would be much cleaner